packages/trader/src/Modules/SmartChart/Components/__tests__/recent-trade-info.spec.tsx

```diff
@@ -0,0 +1,126 @@
+                            tick_count: 5,
+                            tick_stream: [
+                                {
+                                    epoch: 1700203489,
+                                    tick: 2361.33,
+                                    tick_display_value: '2361.33',
+                                },
+                                {
+                                    epoch: 1700203490,
+                                    tick: 2361.3,
+                                    tick_display_value: '2361.30',
+                                },
+                                {
+                                    epoch: 1700203491,
+                                    tick: 2362.1,
+                                    tick_display_value: '2362.10',
+                                },
+                                {
+                                    epoch: 1700203492,
+                                    tick: 2361.69,
+                                    tick_display_value: '2361.69',
+                                },
+                                {
+                                    epoch: 1700203492,
+                                    tick: 2361.69,
+                                    tick_display_value: '2361.69',
+                                },
+                            ],
+                        },
+                    },
+                ],
+            },
+        });
+    });
+
+    it('Should render empty container if contract type is ACCU', () => {
+        mocked_store_props.contract_trade.filtered_contracts[0].contract_info.contract_type = 'ACCU';
+        const { container } = renderComponent({ mocked_store_props });
+        expect(container).toBeEmptyDOMElement();
+    });
+    it('Should render empty container if granularity is more than 0', () => {
+        mocked_store_props.client.is_beta_chart = false;
+        mocked_store_props.contract_trade.granularity = 60;
+        const { container } = renderComponent({ mocked_store_props });
+        expect(container).toBeEmptyDOMElement();
+    });
+    it('Should render empty container after contract is ended', () => {
+        mocked_store_props.contract_trade.filtered_contracts[0].contract_info.status = 'won';
+        const { container } = renderComponent({ mocked_store_props });
+        expect(container).toBeEmptyDOMElement();
+    });
+    it('Should use marker_array if is_beta_chart is false and should render "tick 5/6"', () => {
+        mocked_store_props.client.is_beta_chart = false;
+        mocked_store_props.contract_trade.markers_array[0].contract_info.tick_count = 6;
+        renderComponent({ mocked_store_props });
+        expect(screen.getByText('Tick 5/6')).toBeInTheDocument();
+    });
+    it('Should render "Tick 4/7" contract type is digit contract', () => {
+        mocked_store_props.contract_trade.filtered_contracts[0].contract_info.contract_type = 'digitmatch';
```

Comment:
Does it accept lowerCase? Previously you were passing upperCase

Reply:
regex wants to accept lower case perhaps regex is wrong

Reply:
@henry-deriv regex is correct, it's case-insensitive :) 
Original contract_type coming from API will be 'DIGITMATCH', it would be nice if you could put it in caps to make it closer to real-life case🙏🏻